ui/src/screen.ts

```diff
@@ -0,0 +1,7 @@
+export const lock = (): void => {
+  document.documentElement.classList.add("lock-screen");
```

Comment:
Magical string appears twice, could be put in a const or the style applied directly.

Reply:
I hear you. Not sure it's possible to apply this selector directly: `.lock-screen > *`. Would leave this as a follow up re-factor when we're not under time pressure.